OrbitGl/App.h

```diff
@@ -233,6 +233,14 @@ class OrbitApp final : public DataViewFactory, public CaptureListener {
 
   DataView* GetOrCreateDataView(DataViewType type) override;
 
+  const std::unique_ptr<ProcessManager>& GetProcessManager() {
```

Comment:
Returning (just like passing) a `const std::unique_ptr&` is weird. Return a pointer (if you need to check for the `nullptr` case) or a reference.